jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/SqsUtils.kt

```diff
@@ -50,3 +53,40 @@ fun SqsClient.approximateNumberOfMessages(queueUrl: String): Int? = try {
     getLogger<SqsClient>().error(e) { "SqsClient threw an exception getting approximate number of messages" }
     null
 }
+
+/**
+ * Create a policy statement that allows sending SNS messages to an SQS queue. The Sid
+ * matches how the console does sid (so it won't duplicate it), and the overall policy
+ * matches how the console does it.
+ */
+@Language("JSON")
+fun createSqsSnsSubscribePolicyStatement(sqsArn: String, snsArn: String): String =
+    """
+    {
+        "Sid": "topic-subscription-$snsArn",
+        "Effect": "Allow",
+        "Principal": {
+            "AWS": "*"
+        },
+        "Action": "SQS:SendMessage",
+        "Resource": "$sqsArn",
+        "Condition": {
+            "ArnLike": {
```

Comment:
I assume this just matches the console - but can't this be `ArnEquals` ?

Reply:
This matches the console, it could be `ArnEquals`